resources/scripts/components/server/schedules/ScheduleCheatsheetCards.tsx

```diff
@@ -11,6 +11,10 @@ export default () => {
                         <div css={tw`w-1/2`}>*/5 * * * *</div>
                         <div css={tw`w-1/2`}>every 5 minutes</div>
                     </div>
+                    <div css={tw`flex py-4 px-6 bg-neutral-500`}>
+                        <div css={tw`w-1/2`}>* */1 * * *</div>
```

Comment:
This is a wrong example. Minutes should be 0, or it will execute every minute of the hour, and in this case, every minute.